js/collapse.js

```diff
@@ -102,7 +102,7 @@
 
     var dimension = this.dimension()
 
-    this.$element[dimension](this.$element[dimension]())[0].offsetHeight
+    this.$element[dimension](this.$element[dimension]())[0][$.camelCase(['inner', dimension].join(''))]
```

Comment:
I think this change isn't needed, if I'm correct the `.offsetHeight` is only used to force to trigger a reflow, see http://jsfiddle.net/eo5y271f/2/ for your jsFiddle without any changes to the `collapse.js`